### PR TITLE
add warning to fft re option bit 1

### PR DESCRIPTION
**Pull request by Hwurzburg**
needed to update these in prep for adding per mtr thr based addition

### CODE REVIEW

File: common/source/docs/common-esc-telem-based-notch.rst [hunk 1]
```diff
@@ -16,9 +16,7 @@ If ESCs with telemetry are used (either via a separate telemety wire or bi-direc
...
+.. warning:: If you set the multi-source option (i.e. a notch per motor) mentioned below, the bandwidth should not be half the frequency. It should be greatly reduced as more notches cause more phase lag (i.e. latency), thus the bandwidth needs to be reduced to maintain a reasonable phase lag, else oscillation and a poorer tune will result despite the more accurate filtering compared to throttle-based filtering. The suggested starting point is :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / number of motors. For example, with a quadcopter, :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / 4 is suggested as a starting point.
```
Comment (andyp1per) [inline common/source/docs/common-esc-telem-based-notch.rst:19]: It should be INS_HNTCH_FREQ/4 we have already proved that F/number of motors is in fact wrong in a bad way.

**Review by andyp1per — CHANGES_REQUESTED** [review]
Needs couple of amendments

File: common/source/docs/common-esc-telem-based-notch.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-esc-telem-based-notch.rst:19]: this should be better now, hopefully

Comment (andyp1per) [reply common/source/docs/common-esc-telem-based-notch.rst:19]: I think just take out :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / number of motors - just say starting point should be :ref:`INS_HNTCH_BW<INS_HNTCH_BW>` / 4 - that's always a good starting point and the former is wrong in many instances

**Review by andyp1per — CHANGES_REQUESTED** [review]
Take out number of motors

File: common/source/docs/common-esc-telem-based-notch.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-esc-telem-based-notch.rst:19]: yep...fixed it on esc and forgot to copy it to fft...

**Review by andyp1per — APPROVED** [review]

**Pull request merged**